python/lsst/cp/pipe/ptc.py

```diff
@@ -623,55 +735,76 @@ def calculateLinearityResidualAndLinearizers(self, exposureTimeVector, meanSigna
 
         Returns
         -------
-        c0: `np.float`
+        polynomialLinearizerCoefficients : `list` of `np.float`
```

Comment:
Do not use `np` abbreviation in docstrings. `numpy.float` should be used. Why is this a list of individual numpy floats and not a single numpy array? (and also `float` and `numpy.float` should be identical from your point of view shouldn't they?)